addons/l10n_ch/i18n_extra/it.po

```diff
@@ -328,113 +328,48 @@ msgstr "910 Donazioni, dividendi, compensi, ..."
 #: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Due date</"
-"span><br/>"
-msgstr ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Data di "
-"scadenza</span><br/>"
+msgid "<span class=\"title\">Acceptance point</span>"
+msgstr "<span class=\"title\">Punto di accettazione</span>"
 
 #. module: l10n_ch
 #: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Payable by</"
-"span><br/>"
-msgstr ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Pagabile da</"
-"span><br/>"
+msgid "<span class=\"title\">Account / Payable to</span><br/>"
+msgstr "<span class=\"title\">Conto / Pagabile a</span><br/>"
 
 #. module: l10n_ch
 #: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Reference</"
-"span><br/>"
-msgstr ""
-"<br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Riferimento</"
-"span><br/>"
-
-#. module: l10n_ch
-#: model_terms:ir.ui.view,arch_db:l10n_ch.res_config_settings_view_form
-msgid "<span class=\"o_form_label\">ISR scan line offset</span>"
-msgstr ""
-"<span class=\"o_form_label\">Offset della linea di scansione SRI</span>"
+msgid "<span class=\"title\">Additional information</span><br/>"
+msgstr "<span class=\"title\">Informationi supplementari</span><br/>"
 
 #. module: l10n_ch
 #: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid ""
-"<span class=\"swissqr_text title title_zone\">QR-bill Payment Part</span><br/"
-">"
-msgstr ""
-"<span class=\"swissqr_text title title_zone\">Parte di pagamento QR-fattura</"
-"span><br/>"
+msgid "<span class=\"title\">Amount</span><br/>"
+msgstr "<span class=\"title\">Importo</span><br/>"
 
 #. module: l10n_ch
 #: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid ""
-"<span class=\"swissqr_text title title_zone\">QR-bill Payment Receipt Part</"
-"span><br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Payable to</"
-"span><br/>"
-msgstr ""
-"<span class=\"swissqr_text title title_zone\">QR-fattura Parte Ricevuta di "
-"pagamento</span><br/>\n"
-"\n"
-"                            <span class=\"swissqr_text title\">Pagabile a</"
-"span><br/>"
-
-#. module: l10n_ch
-#: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid "<span class=\"swissqr_text title\">Acceptance point</span>"
-msgstr "<span class=\"swissqr_text title\">Punto di accettazione</span>"
-
-#. module: l10n_ch
-#: model_terms:ir.ui.view,arch_db:l10n_ch.l10n_ch_swissqr_template
-msgid "<span class=\"swissqr_text title\">Additional information</span><br/>"
-msgstr "<span class=\"swissqr_text title\">Ulteriori informazioni</span><br/>"
```

Comment:
@mart-e Is it allowed to remove translations like this in stable? Or should the new stuff just be put next to it?

Reply:
@oco-odoo it's fine for `l10n_*` were translations only applies during the CoA installation. In stable it's problematic to change translatable content because the developer does not translate the new term in all languages on Transifex while merging the PR, here it is translated :)